hc/api/tests/test_list_checks.py

```diff
@@ -34,15 +36,39 @@ def get(self):
     def test_it_works(self):
         r = self.get()
         ### Assert the response status code
+        self.assertEqual(r.status_code, 200)
 
         doc = r.json()
         self.assertTrue("checks" in doc)
 
         checks = {check["name"]: check for check in doc["checks"]}
         ### Assert the expected length of checks
+        self.assertEqual(len(checks), 2)
         ### Assert the checks Alice 1 and Alice 2's timeout, grace, ping_url, status,
         ### last_ping, n_pings and pause_url
 
+        pause_url = reverse('hc-api-pause', args=[self.a1.code])
+        pause_url = settings.SITE_ROOT + pause_url
+        # assertions for Alice 1 -> [a1]
+        self.assertEqual(checks['Alice 1']['timeout'], 3600)
+        self.assertEqual(checks['Alice 1']['grace'], 900)
+        self.assertEqual(checks['Alice 1']['status'], "new")
```

Comment:
Be consistent with the quotation marks you choose to use. Decide between single or double quotation marks. 
I'd prefer double.
This comment applies for all affected files.